### DISCUSSION TITLE
How to create IBOutlet for chat screen in Storyboard for UIView.

### DISCUSSION THREAD

**Discussion by mujtuba11khan** [MessageKit/MessageKit#1665] [Q&A]
I am creating a chat screen from the storyboard and I need to use the chat screen in a uiview created on the storyboard. So my question is what class should I have to assign to Identity Inspector and how can I use MessageKit through a storyboard.

**Comment by Kaspik** [comment]
I haven't used storyboards for 6 years now so I'm not able to help you, sorry. My only recommendation is to not use them in case your project grows bigger, they are not really helpful. :)
